README.md

```diff
@@ -33,8 +33,10 @@ our company's [history and SaaS metrics](https://www.parabol.co/blog/tag/friday-
 ### Prerequisites
 
 - [Node](https://nodejs.org/en/download/)
-- [pnpm](https://pnpm.io/)
-- [Docker Compose](https://docs.docker.com/compose/install/)
+- [PostgreSQL 16.0+](https://www.postgresql.org/) (current specific version available in the [build action](./.github/workflows/build.yml)) with [pgvector plugin](https://github.com/pgvector/pgvector)
+- [Redis 7.0+](https://redis.io) (current specific version available in the [build action](./.github/workflows/build.yml))
+- [pnpm](https://pnpm.io/) (Development only)
```

Comment:
This is needed for building, does that count as development only?

Reply:
You are right and that's a mistake. I'm removing that comment